File: internal/content/testdata/bench/rust/maximize-bounded-score.rs
fn maximize_score(arr: Vec<i64>, k: i64) -> i64 {
    let mut best: i64 = i64::MIN;
    for x in arr {
        let score = (2 * k).div_euclid(1 + 2 * (x - k).abs());
        if score > best {
            best = score;
        }
    }
    best
}

File: internal/content/testdata/bench/java/maximize-bounded-score.java
class Solution {
    public long maximize_score(long[] arr, long k) {
        long best = 0;
        for (long x : arr) {
            long score = (2 * k) / (1 + 2 * Math.abs(x - k));
            if (score > best) best = score;
        }
        return best;
    }
}
